src/components/fellows/index.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import { fetchAutomation } from '../../redux/actions/automation';
+import InfoIcon from '../../assets/icons/Info.svg';
+import onBoarding from '../../assets/icons/vectorpaint.svg';
+import './styles.scss';
+
+export class DeveloperCard extends Component {
+  componentDidMount() {
+    const { fetchAutomations } = this.props;
+    fetchAutomations();
+  }
+
+  renderCards =() => {
+    const { automation: { data } } = this.props;
+    const newData = data.slice(0, 10);
+    const automation = length => length !== 0;
+    const automationMetric = (activities, status) => (
+      _.filter(activities, { status }).length
+    );
+    const metric = (activities, status) => (
+      automation(activities.length) ? automationMetric(activities, status) : 0
+    );
+    return newData.map((card) => {
+      const name = card.fellowName;
+      const newName = _.split(name, ',');
+      const firstInitials = newName[0].charAt(0);
+      const secondInitials = newName[1].charAt(1);
+      return (
+        <div className="card" key={card.id}>
+          <div className="info-cont">
+            <img className="info-icon onBoarding-icon" src={onBoarding} alt="onboarding icon" />
+            <img className="info-icon" src={InfoIcon} alt="info icon" />
+          </div>
+          <div className="developerPicture">
+            <div className="oval-1">
+              {firstInitials}
+              {secondInitials}
+            </div>
+          </div>
+          <div className="developerDetails">
+            <span className="developerName">{card.fellowName}</span>
+          </div>
+          <div className="developerDetails partnerName">
+            <span title={card.partnerName}>{card.partnerName}</span>
+          </div>
+          <div className="developerDetails">
+            <span className="date">1/30/2019 1:41 PM</span>
+          </div>
+          <div className="status-band">
+            <span>
+              {card.freckleAutomations.status.toUpperCase()}
+              <button type="button">RETRY</button>
+            </span>
+          </div>
+          <div className="status-container">
+            <div>
```

Comment:
lines 43 to57 and 59 to 94 keep repeating, you could create this as metadata and import that metadata and map it to generate these elements